podcasts/SceneHelper.swift

```diff
@@ -26,7 +26,8 @@ class SceneHelper {
     class func rootViewController() -> UIViewController? {
         guard !FeatureFlag.newPlayerTransition.enabled else {
             let appScene = connectedScene()?.windows.first(where: { $0.rootViewController is MainTabBarController })
-            return appScene?.rootViewController?.topMostPresentedViewController
+            let rootVC = appScene?.rootViewController
```

Comment:
These changes were not part of this PR, I think this comes from another PR that is still not merged in trunk.